pyproject.toml

```diff
@@ -57,7 +57,7 @@ max-locals = 20
 
 [tool.pytest.ini_options]
 minversion = "6.0"
-addopts = "--benchmark-skip"
+addopts = "--benchmark-skip --durations=50 --cov aiida --cov-report xml --cov-append --verbose"
```

Comment:
The only problem is that it forces every run of pytest to be verbose now and output all the coverage, i.e. when I'm testing locally.
At least for `verbose`, I would rather have this set in the workflow job

Reply:
Ah yeah, that's true. Maybe I move `--cov aiida` and `--verbose` to the actual invocation in `tests.sh`.